src/components/shared/thread/ActivityGraphCanvas.js

```diff
@@ -0,0 +1,229 @@
+// @flow
```

Comment:
Can you please add the license comment at the top of this file?

Reply:
Done